definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState-alt-service-process-server-nonprod.json

```diff
@@ -0,0 +1,65 @@
+[
+  {
+    "LiveFrom": "12/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseStateID": "AwaitingProcessServerService",
+    "UserRole": "caseworker-divorce-courtadmin",
+    "CRUD": "R"
```

Comment:
Can we make this consistent with any decision we come to?

Reply:
Updated